app/models/admin_configuration.rb

```diff
@@ -221,6 +221,20 @@ def self.set_readonly_service_account_permissions(grant_access)
     end
   end
 
+  def self.find_or_create_ws_user_group!
+    groups = Study.firecloud_client.get_user_groups
+    ws_owner_group = groups.detect {|group| group['groupName'] == FireCloudClient::WS_OWNER_GROUP_NAME &&
+        group['role'] == 'Admin'}
+    # create group if not found
+    if ws_owner_group.present?
+      ws_owner_group
+    else
+      # create and return group
+      Study.firecloud_client.create_user_group(FireCloudClient::WS_OWNER_GROUP_NAME)
+      Study.firecloud_client.get_user_group(FireCloudClient::WS_OWNER_GROUP_NAME)
```

Comment:
huh, does the create not return the group itself?  That seem un-RESTful of Terra.

Reply:
Actually, in looking at this, I think that I have a bug still.  The return object for those two endpoints are the same, but it's not the same as the top-level "show me all groups" response.  I need to make an update to account for that structure.  The information is all there, it's just formatted differently.

Reply:
So I was wrong - while the structure is different, the `groupEmail` value is still in the same place, and that's all we need from the response.